third_party/move/move-compiler-v2/tests/reference-safety/write_ref_dest.exp

```diff
@@ -0,0 +1,2 @@
+
+============ bytecode verification succeeded ========
```

Comment:
Add this to transactional tests with paranoid mode enabled.

Reply:
done